.github/workflows/ci-linux-incremental.yml

```diff
@@ -99,56 +99,9 @@ jobs:
          "debian-bullseye",
          "debian-bookworm",
          "fedora-30",
-         "fedora-40",
-         "gentoo-python3.11",
-         "debian-bullseye-i386"]
+         "fedora-40",]
       tox_packages_factors: >-
           ["standard",
            "minimal"]
       docker_push_repository: ghcr.io/${{ github.repository }}/
       max_parallel: 8
-
-  constraints_pkgs-norequirements:
-    needs: [changed_files]
```

Comment:
Please refrain from removing things just because it is broken. They can be fixed later. Turning off is enough.